### DISCUSSION TITLE
Quarto 1.6 – Quarto

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11533] [Blog]
# Quarto 1.6 – Quarto

Quarto 1.6 supports unified branding across formats, updates to RevealJS, a new shortcode to reorder content, a landscape page block, and more. There are also a couple of breaking changes that will affect a small number of users.

https://quarto.org/docs/blog/posts/2024-11-25-1.6-release/

**Comment by G1ngie** [comment]
Was this the version that would fix the GitHub Copilot autocomplete not working in .qmd files? Or should we expect that later?

**Reply by mcanouil** [reply]
This is unrelated to Quarto CLI.
That's a settings set in your IDE (VSCode, Positron) which is set to false in the extension for the IDE.

**Reply by G1ngie** [reply]
Oh man, I am an idiot, sorry! I thought I was reading this was needed to be fixed in an upcoming version of Quarto. I am now realizing it was actually in the RStudio github... I am sorry, I am very new to the github/coding world. Thanks for letting me know!

**Reply by mcanouil** [reply]
In the case of RStudio, that's the IDE itself. I have no idea how copilot is integrated in it.

**Reply by cderv** [reply]
> Oh man, I am an idiot, sorry! I thought I was reading this was needed to be fixed in an upcoming version of Quarto. I am now realizing it was actually in the RStudio github... I am sorry, I am very new to the github/coding world. Thanks for letting me know!

Hey @G1ngie! 😊 No need to be hard on yourself! It's totally okay to post here—seriously, no apologies needed! We all started out as beginners in coding and GitHub at some point. If you ever have questions, just ask away! You'll learn so much by jumping in and figuring things out as you go. Keep it up! 👍
